apps/wallet-mobile/src/TxHistory/ActionsBanner.tsx

```diff
@@ -64,7 +64,7 @@ export const ActionsBanner = ({disabled = false}: {disabled: boolean}) => {
   }
 
   const handleOnSwap = () => {
-    resetTouches()
+    resetSwapForm()
     resetState()
```

Comment:
@banklesss how about lifting up to the "controller" the resetState goes into resetSwapForm?

Reply:
I have checked and the current `resetSwapForm ` calls `resetQuantities` instead of `resetState ` because its related to the form CLEAR action.

Should we have a separate action for each case?

Reply:
`clearSwapForm` vs `resetSwapForm`

Reply:
I added this. Let e know if you think otherwise please.